webapp/src/components/Controls/DatasetSplitToggler.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { fireEvent, screen } from "@testing-library/react";
+import { renderWithRouterAndRedux } from "mocks/utils";
+import DatasetSplitToggler from "./DatasetSplitToggler";
+import { AvailableDatasetSplits, DatasetSplitName } from "types/api";
+
+const handleValueChange = jest.fn();
+const renderDatasetSplitToggler = (
+  availableDatasetSplits: AvailableDatasetSplits,
+  value: DatasetSplitName
+) =>
+  renderWithRouterAndRedux(
+    <DatasetSplitToggler
+      availableDatasetSplits={availableDatasetSplits}
+      value={value}
+      onChange={handleValueChange}
+    />
+  );
+
+describe("DatasetSplitToggler", () => {
+  it("should test the toggle buttons - count, display names and icons of it", () => {
```

Comment:
I find this name is quite confusing. Can we simplify to something like this?
```suggestion
  it("should have two buttons with icons", () => {
```